src/sentry/api/helpers/actionable_items_helper.py

```diff
@@ -0,0 +1,113 @@
+from sentry.models import EventError, PromptsActivity, SourceMapProcessingIssue
+
+fileNameBlocklist = ["@webkit-masked-url"]
+
+priority_ranking = {
+    EventError.INVALID_DATA: 19,
+    EventError.INVALID_ATTRIBUTE: 20,
+    EventError.VALUE_TOO_LONG: 11,
+    EventError.FUTURE_TIMESTAMP: 12,
+    EventError.PAST_TIMESTAMP: 13,
+    EventError.CLOCK_DRIFT: 14,
+    EventError.INVALID_ENVIRONMENT: 15,
+    EventError.SECURITY_VIOLATION: 16,
+    EventError.RESTRICTED_IP: 17,
+    EventError.FETCH_GENERIC_ERROR: 18,
+    EventError.JS_MISSING_SOURCES_CONTENT: 10,
+    EventError.JS_SCRAPING_DISABLED: 9,
+    EventError.NATIVE_BAD_DSYM: 21,
+    EventError.NATIVE_MISSING_OPTIONALLY_BUNDLED_DSYM: 22,
+    EventError.NATIVE_MISSING_DSYM: 23,
+    EventError.PROGUARD_MISSING_MAPPING: 24,
+    EventError.PROGUARD_MISSING_LINENO: 25,
+    SourceMapProcessingIssue.MISSING_RELEASE: 1,
+    SourceMapProcessingIssue.MISSING_SOURCEMAPS: 2,
+    SourceMapProcessingIssue.URL_NOT_VALID: 3,
+    SourceMapProcessingIssue.NO_URL_MATCH: 4,
+    SourceMapProcessingIssue.PARTIAL_MATCH: 5,
+    SourceMapProcessingIssue.DIST_MISMATCH: 6,
+    SourceMapProcessingIssue.SOURCEMAP_NOT_FOUND: 7,
+    SourceMapProcessingIssue.DEBUG_ID_NO_SOURCEMAPS: 8,
+}
+
+errors_to_hide = [
+    EventError.JS_MISSING_SOURCE,
+    EventError.JS_INVALID_SOURCEMAP,
+    EventError.JS_INVALID_SOURCEMAP_LOCATION,
+    EventError.JS_TOO_MANY_REMOTE_SOURCES,
+    EventError.JS_INVALID_SOURCE_ENCODING,
+    EventError.UNKNOWN_ERROR,
+    EventError.MISSING_ATTRIBUTE,
+    EventError.NATIVE_NO_CRASHED_THREAD,
+    EventError.NATIVE_INTERNAL_FAILURE,
+    EventError.NATIVE_MISSING_SYSTEM_DSYM,
+    EventError.NATIVE_MISSING_SYMBOL,
+    EventError.NATIVE_SIMULATOR_FRAME,
+    EventError.NATIVE_UNKNOWN_IMAGE,
+    EventError.NATIVE_SYMBOLICATOR_FAILED,
+]
+
+deprecated_event_errors = [
```

Comment:
i don't believe we are sending these event errors anymore but am checking to confirm